source/upgrade.txt

```diff
@@ -50,6 +50,20 @@ changes between the current and upgrade versions. For example, if you
 are upgrading the driver from v4.0 to v4.5, address all breaking changes from
 the version after v4.0 including any listed under v4.5.
 
+.. _java-breaking-changes-v4.8:
+
+Version 4.8 Breaking Changes
```

Comment:
Though now that I think of it, do we want to call it a breaking change?  It makes it seem like a semantic versioning violation to express it this way. I would prefer something different, though I can't think of quite the right terminology.  Maybe just be more explicit: 

> Version 4.8 Drops Support for EOL server releases

Reply:
Should we update the compat table to remove a checkmark next to all versions < 4.2

Reply:
< 3.4 ?

Reply:
If we say eol server versions, that's everything < 4.2